pkg/engine/keyvaults.go

```diff
@@ -111,19 +120,18 @@ func CreateKeyVaultVMSS(cs *api.ContainerService) map[string]interface{} {
 		},
 	}
 	if useManagedIdentity {
-		dependencies := []string{
-			"[concat('Microsoft.Compute/virtualMachineScaleSets/', variables('masterVMNamePrefix'), 'vmss')]",
-		}
+		dependencies := []string{}
```

Comment:
@DavidParks8 I don't understand this change

Reply:
It is a remnant of when I was testing encryption of the master vmss. It turns out that dependency wasn't needed, and leaving it there created a circular reference where the vmss needed the keyvault, and the keyvault needed the vmss. I didn't see a reason to keep the dependency even after rolling back the master node encryption, because the keyvault can get created regardless of the vmss existence.